Tests/SwiftLintFrameworkTests/RulesTests.swift

```diff
@@ -74,6 +74,10 @@ class RulesTests: XCTestCase {
         verifyRule(DiscouragedObjectLiteralRule.description)
     }
 
+    func testDiscouragedOptionalBoolean() {
```

Comment:
You need to run Sourcery to update the tests list to be run on Linux

Reply:
Eita! I completely forgot to commit include the Linux file.

Reply:
Done :-) Sorry about that, @marcelofabri. Thanks.